postgres/src/test/java/org/jdbi/v3/postgres/TestPostgresTypes.java

```diff
@@ -49,7 +45,18 @@
 public class TestPostgresTypes {
 
     @ClassRule
-    public static JdbiRule postgresDBRule = JdbiRule.embeddedPostgres();
+    public static JdbiRule postgresDBRule = PostgresDbRule.rule(builder -> {
+        // We need to force the locale for the 'testReadWriteMoney' test
+        final String locale;
+
+        if (SystemUtils.IS_OS_WINDOWS) {
+            locale = "English_United States";
+        } else {
+            locale = "en_US";
```

Comment:
This seems to be failing Travis.  Should it be `en_US.UTF-8`?  (That seems to be what it was before).

```
[INFO] Running org.jdbi.v3.postgres.TestPostgresTypes
[log:pid(7708)] INFO init-cd8b89f2-e221-4491-b490-e4b22edbf586:initdb - The files belonging to this database system will be owned by user "travis".
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.067 s <<< FAILURE! - in org.jdbi.v3.postgres.TestPostgresTypes
[ERROR] org.jdbi.v3.postgres.TestPostgresTypes  Time elapsed: 0.067 s  <<< ERROR!
java.lang.IllegalStateException: 
Process [/tmp/embedded-pg/PG-06e3a92a2edb6ddd6dbdf5602d0252ca/bin/initdb, -A, trust, -U, postgres, -D, /tmp/epg8422493978783602566, -E, UTF-8, --locale, en_US] failed
```

Reply:
Yeah that's possible, let me validate this on my laptop.

Reply:
Yup "en_US.UTF-8" fixed it.